src/ui/screens/onboarding/OnboardingDevicesSlide.qml

```diff
@@ -0,0 +1,208 @@
+        Layout.maximumHeight: 152
+        Layout.maximumWidth: 152
+        Layout.alignment: Qt.AlignHCenter
+
+        currentIndex: deviceTypeToggle.selectedIndex
+
+        //Google play store QR code
+        Rectangle {
+            color: MZTheme.theme.white
+
+            Layout.preferredHeight: 152
+            Layout.preferredWidth: 152
+            Layout.maximumHeight: 152
+            Layout.maximumWidth: 152
+
+            MZDropShadow {
+                anchors.fill: parent
+                source: parent
+
+                verticalOffset: 1
+                horizontalOffset: 1
+                color: MZTheme.colors.grey60
+                opacity: .1
+                transparentBorder: true
+                cached: true
+            }
+
+            Image {
+                width: 152
+                height: 152
+
+                //QR coded generated via Adobe Express
+                source: "qrc:/ui/resources/qrcodes/play-store-qrcode.png"
+                fillMode: Image.PreserveAspectFit
+
+                Accessible.role: Accessible.Graphic
+                Accessible.name: MZI18n.OnboardingDevicesSlideQRCodeAndroid
+            }
+        }
+
+        //Apple app store QR code
+        Rectangle {
+            color: MZTheme.theme.white
+
+            Layout.preferredHeight: 152
+            Layout.preferredWidth: 152
+            Layout.maximumHeight: 152
+            Layout.maximumWidth: 152
+
+            MZDropShadow {
+                anchors.fill: parent
+                source: parent
+
+                verticalOffset: 1
+                horizontalOffset: 1
+                color: MZTheme.colors.grey60
+                opacity: .1
+                transparentBorder: true
+                cached: true
+            }
+
+            Image {
+                width: 152
+                height: 152
+
+                //QR coded generated via Adobe Express
+                source: "qrc:/ui/resources/qrcodes/app-store-qrcode.png"
+                fillMode: Image.PreserveAspectFit
+
+                Accessible.role: Accessible.Graphic
+                Accessible.name: MZI18n.OnboardingDevicesSlideQRCodeApple
+            }
+        }
+    }
+
+    Item {
+        Layout.fillHeight: true
+        Layout.minimumHeight: 32
+    }
+
+    MZButton {
+        Layout.leftMargin: MZTheme.theme.windowMargin * 2
+        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+        Layout.fillWidth: true
+
+        width: undefined
+        text: MZI18n.GlobalNext
+
+        onClicked: root.nextClicked()
+    }
+
+    MZLinkButton {
+        Layout.topMargin: 16
+        Layout.leftMargin: MZTheme.theme.windowMargin * 2
+        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+        Layout.bottomMargin: MZTheme.theme.windowMargin
+        Layout.fillWidth: true
+
+        implicitHeight: MZTheme.theme.rowHeight
+
+        labelText: MZI18n.GlobalGoBack
+
+        onClicked: root.backClicked()
```

Comment:
```suggestion
        onClicked: backClicked()
```

I think since `root` is in the same scope you shouldn't need to explicitly call it here

Reply:
Although we would be able to reference `backClicked` without using the parent's id, [QML coding conventions](https://doc.qt.io/qt-6/qml-codingconventions.html#unqualified-access) recommend that we do so